R/gen_tibble.R

```diff
@@ -3,7 +3,17 @@
 #' A `gen_tibble` stores genotypes for individuals in a tidy format. DESCRIBE
 #' here the format
 #'
-#' When loading packedancestry files, missing alleles will be converted from
+#'- *VCF* files: the fast `cpp` parser is used by default. Both `cpp` and `vcfR` parsers
+#' attempt to establish ploidy from the first variant; if that variant is found in a
+#' sex chromosome (or mtDNA), the parser will fail with 'Error: a genotype has more
+#' than max_ploidy alleles...'. To use the fast parser, change the order of variants
```

Comment:
'To parse the vcf file, ' as either parser will then work, not just the fast on.